package.json

```diff
@@ -49,14 +49,13 @@
     "lint:js": "eslint --quiet src __tests__ --ext .js,.jsx",
     "lint:scss": "stylelint \"src/*.scss\"",
     "postbuild": "cd build && npm install --production",
-    "postcommit": "npm run _format",
     "preinstall": "yarn global add nodemon",
     "prepush": "npm run verify",
     "start": "npm-run-all -p watch-css start-js",
     "start-prod": "sudo nodemon ./server.js",
     "start-js": "react-scripts start",
     "test": "react-scripts test --env=jsdom",
-    "verify": "npm run lint && cross-env CI=true npm test",
+    "verify": "npm run fmt && npm run lint && cross-env CI=true npm test",
```

Comment:
I wonder if we should run `lint` before `fmt`. Will `fmt` fail if the code contains a syntax error? Maybe running `lint` first would catch that.